Calathea-Z/ShoeString-Frontend
# corey

- Continue work on @media for larger screens.
- Refactoring UI
- trying new styling
- test
- Continue work on alternate layout for app.
- Feed refactor is complete.
- Build image upload componenet.
- Edit and Delete buttons for main feed are now fully functional. There are two error messages that will need to be dealt with - however they do not seem to be breaking the main functionality. Will need to be returned to later.
- Working code in codition to merge with other Zach development branch to update styling.
- clearing errors, commenting out unused imports and variables
- Switching branches
- setting up dev database config, testing CreatePost route
- Readjust formatting of feed for new UI.
- Continue work on image upload.
- Created UploadWidget.js component which is functionally. It allows for a modal to pop up on the screen and the user to upload a photo to the cloudinary cloud db
- Clean up code in CreatPost.jsx.
- Photo upload with no widget is functional to Cloudinary. I am able to retrieve the url for said photo. I am hitting a blocker on setting that in state.
- save
- Working on styling
- Working on styling
- Continue formatting work.
- edit styles add profile iamge
- Continue style work.
- added latitude and longitude fields to CreatePost form, then add them to newForm.location AKA post.location | added comma and space between location array fields in post display
- Styling finished for header and footer. Uploading for merge to main to check on physical phone.
- added favicon, changed window name from React App to ShoeString
- Continue styling work.
- Test
- updated Planning.md and Readme.md
- Fixing styling after merge conflict
- Refactoring styling
- Fix post button.
- Clean up code in CreatPost.jsx.
- Working modal base.
- Modal is fully functional. Added logic from creatPost.jsx into the modal.
- Added search bar and functions to find username ID from search -- appears in console log, but late night coding brain is out of juice for the next step
- Clean up code. Remove original creatPost.jsx file and add into modal. Change name of modal to createPost.jsx.
- Page refreshes on update.
- Clean up code.
- Update profile logo.
- Fixed a styling issue with like button where the heart was bleeding over onto the photo.
- Conditional Rendering for EDIT and DELETE functions - user must login via auth (click user icon to nav to auth) | basic styling for auth register/login page
- removed dev borders in styling
